src/main/java/org/grizz/model/Building.java

```diff
@@ -0,0 +1,15 @@
+package org.grizz.model;
+
+import lombok.Builder;
+import lombok.Data;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Data
+@Builder
+@Document(collection = "buildings")
```

Comment:
Buildings should not be stored in mongo on it's own. Building cannot exist without planet so lets keep all buildings inside plannets collection as a part of planet.